dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -68,4 +76,24 @@ def buffer_abuse_score(score = -50)
       'rack.input' => StringIO.new
     )
   end
+
+  def freeze_featured_project(project_id)
+    project = Project.find_by(id: project_id)
+    return if project.nil?
+    project_value = JSON.parse(project.value)
+    project_value["frozen"] = true
+    project_value["updatedAt"] = DateTime.now.to_s
+    project.update! value: project_value.to_json
+  end
+
+  def unfreeze_featured_project(project_id)
+    project = Project.find_by(id: project_id)
+    return if project.nil?
+    project_value = JSON.parse(project.value)
+    project_value["frozen"] = false
+    # Unhide in case this project was frozen manually.
```

Comment:
Also think this comment might be worth expanding on for future readers.

Reply:
Updated!